### PR TITLE
new machine learning model integreted

### PR DESCRIPTION
**Pull request by florpadilla**
Hi guys! Please, take a look to the model and the changes added.

### CODE REVIEW

File: MACHINE-LEARNING-SERVICE/src/controllers/objectRecognition.controller.js [hunk 1]
```diff
@@ -10,35 +10,29 @@ accordance with the terms of the license agreement you entered into
...
-    const chooseModel = await HandlerModel.chooseModel(model, object, percentage);
-    res.send(chooseModel);
-    
-    }catch(error){
+      res.send(await ModelFactory.giveResult(model, object, percentage));
+    } catch (error) {
       res.status(error.status).send({
         Error: error.message,
-        Code: error.code
+        Code: error.code,
```
Comment (stospina) [inline MACHINE-LEARNING-SERVICE/src/controllers/objectRecognition.controller.js]: In line 35 the comma is not necessary

File: MACHINE-LEARNING-SERVICE/src/helpers/filterResults.helper.js [hunk 2]
```diff
@@ -21,14 +21,14 @@ class FilterResults {
...
-          prediction.score >= percentage  || prediction.probability >= percentage
-        ) {
+          if ((String(prediction.className).includes(objectRequired) || String(prediction.class).includes(objectRequired) ) &&
+          (prediction.score >= percentage || prediction.probability >= percentage)
+          ) {
           foundObjectsArray.push({
-            fileName: predictions.fileName,
+            Second: predictions.fileName.split('.')[0],
             object: prediction.class || prediction.className ,
             Score: prediction.score  || prediction.probability ,
```
Comment (stospina) [inline MACHINE-LEARNING-SERVICE/src/helpers/filterResults.helper.js]: The comma can be omitted

**Review by stospina — COMMENTED** [review]
Look good

File: MACHINE-LEARNING-SERVICE/src/helpers/filterResults.helper.js (hunk 2 above)
Comment (florpadilla) [reply MACHINE-LEARNING-SERVICE/src/helpers/filterResults.helper.js]: thanks! I just fixed it

File: MACHINE-LEARNING-SERVICE/src/controllers/objectRecognition.controller.js (hunk 1 above)
Comment (florpadilla) [reply MACHINE-LEARNING-SERVICE/src/controllers/objectRecognition.controller.js]: Fixed! Thanks!!

**Pull request merged**